core/src/main/kotlin/com/bumble/appyx/core/navigation/Operation.kt

```diff
@@ -8,6 +8,57 @@ interface Operation<NavTarget, State> :
 
     fun isApplicable(elements: NavElements<NavTarget, State>): Boolean
 
+    /**
+     * Transitions all elements to a new state
+     */
+    fun NavElements<NavTarget, State>.transitionTo(
+        newTargetState: State
+    ) = transitionToIndexed(newTargetState) { _, _ -> true }
```

Comment:
This creates an unnecessary object, and also doesn't need to access the index. I would recommend just mapping and calling

```
element.transitionTo(
                newTargetState = newTargetState,
                operation = this@Operation
            )
```

It might be slightly more repetitive code wise, but it's better for the end user